src/App.jsx

```diff
@@ -1,17 +1,31 @@
 import React from 'react';
 import { Route, Routes, BrowserRouter as Router } from 'react-router-dom';
-import HomePage from './pages/HomePage';
-import LoginPage from './pages/LoginPage';
+import HomePage from './pages/HomePage/HomePage';
+import LoginPage from './pages/Login/LoginPage';
```

Comment:
let's rename the component from `LoginPage` to `Login` so that it follows the same name as the folder